### DISCUSSION TITLE
NEM 5 minute market price

### DISCUSSION THREAD

**Discussion by charlesesson** [opennem/opennem#238] [General] [answered]
Hi

I am looking for an API that allows me to access the last 5 minute market price.
Is this project suitable? If so I assume there is a server someone to query.
Is there some means to contribute to the server cost if I use the service.

Regards
Charles Esson

**Comment by nc9** [comment, marked as answer]
Hi Charles,

We sure do have that, the API docs are at:

https://api.opennem.org.au/docs

You will find the price information at `/stats/price` endpoint. 

Could you send me an email on nik at opennem.org.au? We're in the process of setting up authentication for the API and we're making an attempt to begin to register API users and support them with their use cases. Thanks!

**Comment by charlesesson** [comment]
Hi 
I'm sure it is there but I can't can't find contact information on the opennem.org.au site. It lands on the page: 
https://opennem.org.au/energy/nem/?range=7d&interval=30m.  ( Very nice piece of work.)
I have looked around and note you have had a pretty interesting life.
My email is charles.esson@gmail.com
Regards
